core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/plangen/LogicalPlan.java

```diff
@@ -517,7 +519,7 @@ private void handleSetInputOperator(IOperator src, IOperator dest) throws PlanGe
      
     /*
      * Finds the sink operator in the operator graph.
-     * 
+     *
      * This function assumes that the graph is valid and there is only one sink in the graph.
      */
     private ISink findSinkOperator(HashMap<String, IOperator> operatorObjectMap) throws PlanGenException {
```

Comment:
This `findSinkOperator` function is no longer called anywhere. remove it or merge it with `findSinkOperators` properly.

Reply:
I will remove it.